api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1446,6 +1446,32 @@ message DeleteSAMLIdPServiceProviderRequest {
   string Name = 1 [(gogoproto.jsontag) = "name"];
 }
 
+// CreatePluginRequest creates a new plugin from the given spec and initial credentials.
+message CreatePluginRequest {
+  // Plugin is the plugin object without live credentials.
+  types.PluginV1 Plugin = 1 [(gogoproto.jsontag) = "plugin"];
+  // InitialCredentials are the initial credentials of the plugin.
+  // In the scope of processing this request, these are exchanged for
+  // "live" credentials, which are stored in the Plugin.
+  types.PluginInitialCredentialsV1 InitialCredentials = 2 [(gogoproto.jsontag) = "initial_credentials"];
+}
+
+// SetPluginCredentialsRequest is a request to set credentials for an existing plugin
+message SetPluginCredentialsRequest {
+  // Name is the name of the plugin instance.
+  string Name = 1 [(gogoproto.jsontag) = "name"];
+  // Credentials are the live plugin credentials.
```

Comment:
Maybe adding something more clear such as:
`are the credentials obtained after exchanging the bootstrap credentials and successive renewals`